controller/articles.controllers.js

```diff
@@ -0,0 +1,18 @@
+const selectArticleById = require("../model/articles.models");
+
+function getArticlesById(request, response, next) {
+  const articleId = request.params.article_id;
+
+  selectArticleById(articleId)
+    .then((article) => {
+      if (article.rows.length === 0) {
+        return Promise.reject({ status: 404, msg: "id not found" });
+      }
+      response.status(200).send(article.rows[0]);
```

Comment:
To fit with convention, this should be sent back as an object from the controller on an appropriately-named key. This makes your API easier for other developers to understand ( e.g. { api } or { topics })